web_01.html

```diff
@@ -169,7 +167,7 @@ <h1>Vectorized Ideas</h1>
                         </div>
                         <div class="project__content">
                             <a href="#" class="project__image">
-                                <img src="/images/logo_mockup.jpg" alt="">    
+                                <img src="/images/logo_mockup.jpg"></img>    
```

Comment:
`img` tag doesn't have children and it is a self-closing tag similar to `<hr />`, `<link />` and etc hence it should look like this: `<img />`